src/components/InfoPopup/InfoPopup.css

```diff
@@ -1,80 +1,79 @@
 .popup {
-  display: flex;
```

Comment:
```jsx

<PopupWithForm
          handleLogin={handleLogin}
          loggedIn={loggedIn}
          onOpen={openRegisterPopup}
          submitHandler={handleLogin}
          isOpen={isSignInPopupOpen}
          onClose={closeAllPopups}
          name="sign-in"
          title="Sign in"
          buttonText="Sign in"
          linkText="Sign up"
        >
          <PopupInput
            title="Email"
            id="input_type_email"
            type="email"
            placeholder="Enter email"
            name="Email"
          />
          <PopupInput
            title="Password"
            id="input_type_password"
            type="password"
            placeholder="Enter password"
            name="Password"
          />
        </PopupWithForm>

```

* You need to have `Login` and `Register` components in your app. So, please put the markup of the popups inside them and use only `Login` and `Register` in component `App`


so, this markup will be in `Login`

Reply:
```jsx

<PopupWithForm
          onSignInPopupClick={handleFormClick}
          submitHandler={handleRegister}
          isOpen={isSignUpPopupOpen}
          onOpen={openLoginPopup}
          onClose={closeAllPopups}
          name="sign-up"
          title="Sign up"
          buttonText="Sign up"
          linkText="Sign in"
        >
          <PopupInput
            title="Email"
            id="input_type_email"
            type="email"
            placeholder="Enter email"
            name="Email"
          />
          <PopupInput
            title="Password"
            id="input_type_password"
            type="password"
            placeholder="Enter password"
            name="Password"
          />
          <PopupInput
            title="Username"
            id="input_type_username"
            type="text"
            placeholder="Enter your username"
            name="Username"
          />
        </PopupWithForm>

```


and this markup will be in `Register`